### PR TITLE
Nutanix CSI driver v2.2.0 release

### PR DESCRIPTION
**Pull request by subodh01**

### CODE REVIEW

File: deploy/Centos/csi-driver.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: storage.k8s.io/v1beta1
 kind: CSIDriver
 metadata:
-  name: com.nutanix.csi
+  name: csi.nuatnix.com
```
Comment (tuxtof) [inline deploy/Centos/csi-driver.yaml]: you have nuatnix everywhere

**Comment by tuxtof** [comment]
@subodh01 can you also move the csi-plugin/deploy/Centos/crd directory to csi-plugin/deploy because it is common to each distribution

**Comment by tuxtof** [comment]
what is the upgrade impact on existing sc / pvc / pv , passing from com.nutanix.csi to csi.nutanix.com ? 
do we have an upgrade procedure ?

File: deploy/Centos/csi-driver.yaml (hunk 1 above)
Comment (subodh01) [reply deploy/Centos/csi-driver.yaml]: Fixed it, thanks

**Comment by subodh01** [comment]
> @subodh01 can you also move the csi-plugin/deploy/Centos/crd directory to csi-plugin/deploy because it is common to each distribution

Done

**Comment by subodh01** [comment]
> what is the upgrade impact on existing sc / pvc / pv , passing from com.nutanix.csi to csi.nutanix.com ?
> do we have an upgrade procedure ?

This will require migrating existing PV to new driver name format. We will look at providing a tool or script for that in future. As data is involved, I won't recommend migrating. CSI driver name change has no impact in functionality.

**Review by sourav-anand1 — APPROVED** [review]
LGTM

**Comment by sourav-anand1** [comment]
@subodh01  Now that we have 2 quick start guides, should we move them into a folder? it will be a bit more organized then.

File: deploy/Ubuntu1804/ntnx-csi-provisioner.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ spec:
       - name: csi-provisioner
         image: quay.io/k8scsi/csi-provisioner:v1.3.0
```
Comment (tuxtof) [inline deploy/Ubuntu1804/ntnx-csi-provisioner.yaml]: @subodh01 why we don't use the same csi-provisioner tag than in Centos example

and same for csi-resizer in line 39

and csi-snapshotter containers is not in ubuntu example ??

and snapshot permissions is not present in ubuntu rbac file

File: deploy/SLES15SP1/ntnx-csi-provisioner.yaml [hunk 3]
```diff
@@ -24,7 +24,7 @@ spec:
       - name: csi-provisioner
         image: quay.io/k8scsi/csi-provisioner:v1.3.0
```
Comment (tuxtof) [inline deploy/SLES15SP1/ntnx-csi-provisioner.yaml]: same for SLES file

csi-provisioner and csi-resizer tag are not the same as centos deploy example

and all snapshot part is missing

**Comment by tuxtof** [comment]
Another comment

we implement PSP in SLES exemple why not leverage same approach for other distribution ?

**Comment by tuxtof** [comment]
we also need to add lvm prerequisite in OS dependency

for exemple on CentOS you need the lvm2 package

**Comment by subodh01** [comment]
> @subodh01 Now that we have 2 quick start guides, should we move them into a folder? it will be a bit more organized then.

We will do in the next release.

File: deploy/SLES15SP1/ntnx-csi-provisioner.yaml (hunk 3 above)
Comment (subodh01) [reply deploy/SLES15SP1/ntnx-csi-provisioner.yaml]: Updated csi-provisioner and csi-resizer versions. I will leave it to SUSE team to test and add support for snapshot.

File: deploy/Ubuntu1804/ntnx-csi-provisioner.yaml (hunk 2 above)
Comment (subodh01) [reply deploy/Ubuntu1804/ntnx-csi-provisioner.yaml]: done

**Comment by subodh01** [comment]
> Another comment
> 
> we implement PSP in SLES exemple why not leverage same approach for other distribution ?

I will test and include it next update.

**Comment by subodh01** [comment]
> we also need to add lvm prerequisite in OS dependency
> 
> for exemple on CentOS you need the lvm2 package

Added a comment in quick start guide.

**Pull request merged**